tests/aws/services/stepfunctions/templates/credentials/credentials_templates.py

```diff
@@ -0,0 +1,37 @@
+import os
+from typing import Final
+
+from tests.aws.services.stepfunctions.templates.template_loader import TemplateLoader
+
+_THIS_FOLDER: Final[str] = os.path.dirname(os.path.realpath(__file__))
+
+
+class CredentialsTemplates(TemplateLoader):
```

Comment:
Wanna just clarify that we will only ever need the `Credentials` field when doing cross-account integrations? (see [docs](https://docs.aws.amazon.com/step-functions/latest/apireference/API_TaskCredentials.html))